app/lib/sortByLocalType.js

```diff
@@ -0,0 +1,24 @@
+const localOrder = {
+  City: 5,
```

Comment:
What is the likelihood these will change?
If they did change or another one was added the comparisons on lines [14](https://github.com/nhsuk/connecting-to-services/pull/369/files#diff-6f5c92439fec630a69c6d2ecf8cb158fR14) and [17](https://github.com/nhsuk/connecting-to-services/pull/369/files#diff-6f5c92439fec630a69c6d2ecf8cb158fR17) are going to end up making that new `local_type` be smaller than anything else. And we aren't going to know about it.
It would be good to have something logged so that we could alert on such an event happening.

Reply:
I'll log an error on 'undefined' from the lookup to ensure we are aware of new codes being introduced.

Reply:
I think the order of these needs tweaking. See this [codelist](https://www.ordnancesurvey.co.uk/xml/codelists/LocalType.xml). Moving `Surburban Area` down to the lowest level would mean it gets sorted below the more easily well defined places and would be sorted alongside the other area type value of `Other Settlement`.